src/raft.c

```diff
@@ -543,6 +543,7 @@ static void executeLogEntry(RedisRaftCtx *rr, raft_entry_t *entry, raft_index_t
                                              entry->data_len) != RR_OK) {
             PANIC("Invalid Raft entry");
         }
+        tmp.client_id = entry->session;
```

Comment:
I think we should stick to a single variable name :) 

We use `client_id`, `session` and `session_id` in the PR. It might be confusing in future. If you think `client_id` is more accurate, we can even change in libraft as well if you want.

Reply:
I think session is more accurate in the context of libraft, I think client_id might be more accurate in the context of Redis.  i.e. RedisRaft would store client_id as libraft's session.

I'm not stuck to this idea, but that's the reason for it.  we can discuss in our daily.